modules/Behaviour/behaviour_manage_addMultiProcess.php

```diff
@@ -41,18 +41,18 @@
     } else {
         $gibbonPersonIDMulti = null;
     }
-    $date = $_POST['date'];
-    $type = $_POST['type'];
+    $date = $_POST['date'] ?? '';
+    $type = $_POST['type'] ?? '';
     $descriptor = null;
     if (isset($_POST['descriptor'])) {
         $descriptor = $_POST['descriptor'];
     }
     $level = null;
     if (isset($_POST['level'])) {
-        $level = $_POST['level'];
+        $level = $_POST['level'] ?? '';
     }
```

Comment:
With null coalesce, there is no need to wrap the assignment in a if-then-isset check anymore. This is the same as:
```php
$level = $_POST['level'] ?? null;
```

Reply:
Thanks for the feedback! I'm still quite new to refactoring, so I appreciate the advice. - Harry